plugin.rb

```diff
@@ -36,6 +39,21 @@
   add_to_serializer(:user, :reminders_frequency) do
     RemindAssignsFrequencySiteSettings.values
   end
+
+  add_to_serializer(:group_user, :assignments_count) do
+    if scope.can_assign?
+      Topic.joins("JOIN topic_custom_fields tcf ON topics.id = tcf.topic_id AND tcf.name = 'assigned_to_id' AND tcf.value IS NOT NULL")
+        .where("tcf.value = ?", object.id.to_s).count
+    end
+  end
+
+  add_to_serializer(:group_show, :assignment_count) do
```

Comment:
We still need to secure this. It should only be visible to users who `can_assign`